### ISSUE TITLE
Plotly.js cluster not working correctly

### ISSUE THREAD

**Issue by Hyron48** [plotly/plotly.js#6948]
Hello everyone. I'm using Plotly.js on angular to plot points on a scattermapbox. However, when I enable clusters, the call to get the fonts for the clusters fails. however I have no way to change the font URL. by any chance can you help me? Unfortunately on codepen I tried to recreate the case, but it seems that the cluster is not enabled at all.

[codepen link](https://codepen.io/Giuseppe-Spina/pen/eYoywYW)

Error image:
<img width="809" alt="Screenshot 2024-04-04 alle 10 42 38" src="https://github.com/plotly/plotly.js/assets/73824388/492a8266-ce1e-4b68-8840-e75f72b1c018">

**Comment by TillerBurr** [comment]
This seems very similar to a bug I fixed previously: https://github.com/plotly/plotly.js/pull/6652. Is the code in the codepen the same that was causing the error?

**Comment by TillerBurr** [comment]
Also, I just want to confirm that you are using the latest version.

**Comment by Hyron48** [comment]
Yes, I'm using the latest version of Plotly and the code on codepen is the same as mine, but I don't understand why on my project it returns the error, but on codepen the cluster doesn't work at all

**Comment by emilykl** [comment]
@Hyron48 The codepen link leads to a bar chart so I'm not able to verify the bug. I'm going to close this issue since we believe the problem was fixed in #6652, but feel free to open a new issue with reproduction steps if you're still seeing the bug.
